data/reporter.js

```diff
@@ -53,7 +53,13 @@ var initialize = function(data) {
 
         var e10s = document.createElement("div");
         e10s.setAttribute("class", "version");
-        e10s.appendChild(document.createTextNode((document.addonReports[i].multiprocessCompatible ? "Compatible" : "Not compatible") + " with multi-process."));
+        var e10s_link = document.createElement("a");
+        e10s_link.setAttribute("href", "#");
+        e10s_link.addEventListener("click", function() { self.port.emit("openE10sIntroLink"); });
```

Comment:
Can you put the contents of the function on the next line? It makes it easier to read.

```js
e10s_link.addEventListener("click", function() {
    self.port.emit("openE10sIntroLink");
});
```